diagnostics-extension/src/services/telemetry.ts

```diff
@@ -76,6 +76,9 @@ export class TelemetryServiceImpl extends TelemetryService {
   async getOemData(): Promise<OemData> {
     return dpsl.telemetry.getOemData();
   }
+  async getCpuInfo(): Promise<CpuInfo> {
```

Comment:
Curious, @1c3t3a is that expected that we didn't have APIs in example extensions?

Reply:
I see. Can I assume that this is just an example and we expect new users follow this example to add missing pieces by themselves. Then, I don't think we need to keep this example up to date with latest API.